docs/tools/replace.sh

```diff
@@ -107,8 +195,7 @@ function main() {
         pod_namespace=$(get_pod_namespace "${volume}")
 
         if ! kubectl delete pod "${pod_name}" --namespace "${pod_namespace}"; then
-            echo "unable to delete pod ${pod_name} using volume ${volume}"
-            exit 1
+            echo "unable to delete pod '${pod_name}' using volume '${volume}'"
```

Comment:
You could change to `echo "unable to delete pod '${pod_name}' using volume '${volume}'; please restart the pod manually`